src/concepts/glean.md

```diff
@@ -0,0 +1,72 @@
+
+Contents:
+
+*   [Overview](#overview)
+*   [What does it offer](#what-does-it-offer)
+*   [How to use Glean](#how-to-use-glean)
+*   [Contact](#contact)
+*   [References](#references)
+
+# Overview
+
+Glean consists of different pieces:
+
+*   Product-side tools - [The Glean SDK](https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/README.md) is what products integrate and record data into.
+*   Services - this is where the data is stored and made available for analysis in our data platform.
+*   Data Tools - these are used to look at the data, performing analysis and setting up dashboards.
+
+![drawing](../assets/Glean_overview.jpg)
+
+# What does it offer
+
+Glean is designed to support typical product analytics use-cases and encourage best practices by requiring clearly defined metrics through the following:
+
+**Basic product analytics are collected out-of-the-box in a standardized way.**
+A baseline of analysis is important for all our mobile application, from counting active users to retention and session times. This is supported out-of-the-box by the library and work consistently across our mobile products.
+
+**No custom code is required for adding new metrics to a product.**
+To make engineers more productive, the SDK keeps the amount of instrumentation code required for metrics as small as possible. Engineers only need to specify what they want to instrument, with which semantics and then record the data using the Glean SDK. The SDK takes care of storing & sending that data reliably.
+
+**Following [lean data practices](https://leandatapractices.com/) through SDK design choices.**
+It's easy to limit data to what's necessary and documentation can be generated easily, aiding both transparency & understanding for analysis.
+
+**Better data tooling integration due to standardized data types & registering them in machine-readable files.**
+When having collected data described in machine-readable files, our various data tools can read them and support metrics automatically, without manual work.
+
+**Due to common high-level concepts for metrics, APIs & data tools can better match the use-cases.**
+To make the choice easier for which metric type to use, we are introducing higher-level data types that offer clear and understandable semantics - when you want to count something, you use the _"count"_ type. This also gives us opportunities to offer better tooling for the data, both on the client and for data tooling.
+
+**Basic semantics on how the data is collected are clearly defined by the library.**
+To make it easier to understand the general semantics of our data, the Telemetry library will define and document when which kind of data will get sent. This gives data analysis common basic semantics.
```

Comment:
Telemetry -> Glean?

"when which kind of data" -> "when and what/which kind of data" ?

Reply:
That second suggestion sounds harder to understand/parse to me, dropping it.